lms/djangoapps/teams/static/teams/js/views/team_actions.js

```diff
@@ -0,0 +1,50 @@
+;(function (define) {
+    'use strict';
+    define([
+        'gettext',
+        'backbone',
+        'text!teams/templates/team-actions.underscore'
+    ], function (gettext, Backbone, team_actions_template) {
+        return Backbone.View.extend({
+            events: {
+                'click a.browse-teams': 'browseTeams',
+                'click a.search-team-descriptions': 'searchTeamDescriptions',
+                'click a.create-team': 'showCreateTeamForm'
+            },
+
+            initialize: function (options) {
+                this.template = _.template(team_actions_template);
+                this.teamParams = options.teamParams;
+            },
+
+            render: function () {
+                var message = interpolate_text(
+                    gettext("Try {browse_all_teams} or {search_team_descriptions}. If you still can't find a team to join, {create_new_team}."),
```

Comment:
And here's a link to the translation docs: https://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#don-t-append-strings-interpolate-values. Again, if you need help please reach out directly to me.